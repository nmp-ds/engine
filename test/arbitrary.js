import { setup } from "./_helpers.js";
import { expect, test } from "vitest";


setup();

test('arbitrary', async ({ uno }) => {
  const classes = ['[--w-content: "I am awesome"]', '[--f-modal-max-height: 80%]'];

  const { css } = await uno.generate(classes);
  expect(css).toMatchSnapshot();
});

