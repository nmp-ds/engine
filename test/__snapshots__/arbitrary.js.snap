// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`arbitrary 1`] = `
"/* layer: default */
.\\\\[--f-modal-max-height\\\\:\\\\ 80\\\\%\\\\]{--f-modal-max-height:80%;}
.\\\\[--w-content\\\\:\\\\ \\\\\\"I\\\\ am\\\\ awesome\\\\\\"\\\\]{--w-content:\\"I am awesome\\";}"
`;
